.dev/docker/entrypoint.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/ash
+## Ensure we are in /app
+
+cd /app
+
+## check for .env file and generate app keys if missing
+if [ -f /app/var/.env ]; then
+  echo "external vars exist"
+  rm /app/.env
+
+  ln -s /app/var/.env /app/
```

Comment:
I am running an earlier version of the dockerized panel that has a very similar entrypoint.sh file and have a small suggestion to make. This line will cause the container to fail to restart after you have created and run the container once before, my fix was to simply add a `-f` flag to the command to force it to update.

One additional question, was having Lets Encrypt support out of the box not a goal for the container?

Reply:
add a `-`f to what line?

Also I was looking at auto let's encrypt but haen't worked it yet. My older images were using caddy, which has auto ssl.

Reply:
Oops sorry, thought I had highlighted a single line. The change would be on line 11 and change it to `ln -sf /app/var/.env /app/` otherwise it tries to create the symlink to the file and errors out because it had already been created from the first the entrypoint.sh script had been run.

I had submitted a PR that added a dockerfile that add's Let's Encrypt support https://github.com/pterodactyl/panel/pull/1117 . Most of the changes were done in the Dockerfile, entrypoint.sh, and the README.

Reply:
I agree with @quikstrike on the `-f`, that screws me up quite frequently with scripted deployments.

Reply:
There is nothing for it to force overwrite. The container doesn't have that file to mess up linking it. It shouldn't be an issue.